public/js/index.js

```diff
@@ -77,3 +77,12 @@ function onGoogleLoad() {
     getYoutubeFeed(youtubeCINPlaylistID, 15, "#cin-videos");
   }
 }
+
+//Comments Character Count
+comments.onkeyup = function(){
```

Comment:
Also, you probably meant to use `document.getElementById("comments") here

Reply:
yeah noticed that after